warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ContentIndexingColumnBasedHandler.java

```diff
@@ -643,4 +660,69 @@ protected BloomFilterUtil newBloomFilterUtil(final Configuration configuration)
         return util;
     }
     
+    protected void indexListEntries(final NormalizedContentInterface nci, boolean indexField, boolean reverseIndexField, StatusReporter reporter) {
+        
+        if (!(indexField || reverseIndexField)) {
+            return;
+        }
+        
+        String indexedFieldName = nci.getIndexedFieldName();
+        String modifiedFieldName = indexedFieldName + listFieldNameSuffix;
+        String content = nci.getIndexedFieldValue();
+        
+        String[] tokens = StringUtils.split(content, listDelimiter);
+        
+        for (String token : tokens) {
+            
+            // Track the number of tokens processed
+            counters.increment(ContentIndexCounters.ORIGINAL_PROCESSED_COUNTER, reporter);
+            
+            Set<NormalizedContentInterface> normalizedValueFields;
+            if (indexField) {
+                NormalizedContentInterface newField;
+                
+                newField = (NormalizedContentInterface) (nci.clone());
+                newField.setFieldName(modifiedFieldName);
+                // don't put tokens in the event.
+                newField.setEventFieldValue(null);
+                newField.setIndexedFieldValue(token);
+                
+                // normalize the value
+                normalizedValueFields = contentHelper.normalizeFieldValue(modifiedFieldName, newField, true);
+                if (CollectionUtils.isNotEmpty(normalizedValueFields)) {
+                    index.putAll(modifiedFieldName, normalizedValueFields);
+                    // add this token to the event fields so a
+                    // local fi\x00 key gets created
+                    // NOTE: we already assigned it to the
+                    // 'indexOnly' list so it won't show up in
+                    // the event
```

Comment:
indexOnly?  Why to these list indexed fields need to be indexOnly?

Reply:
This is similar to tokenization. The individual values are of no interest when retrieving the record. Only the list of values are of interest. But to search for a list that contains a specific (or range) of value(s) needs to have those entries indexed.